fix: verify now supports fully qualified image names, would error prior

aaron-prindle opened the pull request:
Fixes #7570

tejal29 reviewed (commented):
Can we add some tests. I am not sure why path.Base fixes this issue.

aaron-prindle commented:
The issue was that image names were being used as container names when called for images that skaffold did not build.  docker has a validation regexp on container names that image names do not meet (no "/" for example, etc.).  I have updated the existing integration tests to use a non-skaffold built image with a fully qualified path `docker.io/hello-world:latest` to trigger the current error which is now fixed w/ this PR
